conftest.py

```diff
@@ -144,3 +144,26 @@ def teardown_es():
         teardown_es()
     else:
         yield
+
+@pytest.fixture(scope='function', autouse=True)
+def _mock_share_marker(request, es6_client):
+    """Clear out all indices and index templates before and after
+    tests marked with ``es``.
+    """
+    marker = request.node.get_closest_marker('mock_share')
```

Comment:
Yes, multiple fixtures is probably better. I just want to use `responses` to get away from selectively mocking functions to avoid hitting SHARE, that can be kind of wonky.

Reply:
[looks like you could mock multiple responses in one fixture](https://github.com/getsentry/responses#responses-as-a-pytest-fixture), if that helps.

i'm pro-`responses`, seems much cleaner than selectively mocking functions, i agree 👍